charts/shoot-core/components/charts/vpn-shoot/templates/psp/vpn-shoot-psp.yaml

```
@@ -1,19 +1,8 @@
+# TODO(shafeeqes): Delete this Helm chart in a future version.
+---
 apiVersion: {{ include "podsecuritypolicyversion" .}}
 kind: PodSecurityPolicy
 metadata:
   name: gardener.kube-system.vpn-shoot
-spec:
-  privileged: true
-  volumes:
-  - secret
-  - emptyDir
-  allowedCapabilities:
-  - NET_ADMIN
-  runAsUser:
-    rule: RunAsAny
-  seLinux:
-    rule: RunAsAny
-  supplementalGroups:
-    rule: RunAsAny
-  fsGroup:
-    rule: RunAsAny
+  annotations:
+    resources.gardener.cloud/mode: Ignore
```

Can you leave 1 empty line at the end of the file?

```suggestion
    resources.gardener.cloud/mode: Ignore

```